packages/main/src/TimePicker.ts

```diff
@@ -116,7 +116,9 @@ class TimePicker extends TimePickerBase {
 	formatPattern!: string;
 
 	onBeforeRendering() {
-		this.value = this.normalizeValue(this.value!) || this.value;
+		if (this.value) {
+			this.value = this.normalizeValue(this.value!) || this.value;
```

Comment:
We dont need the assertion here. Linter would complain. It didnt complained on sbmit, because the local server was most likely on.

Reply:
Yes, it complained on local as well, fixed it but i guess it wasnt part of the commit, fixing it.